src/commands/instances/create.ts

```diff
@@ -38,11 +43,17 @@ export default class CreateCommand extends Command {
         name,
         description,
         integration,
+        version,
         customer,
         "config-vars": configVars,
       },
     } = await this.parse(CreateCommand);
 
+    if (!integration && !version) {
+      this.error('Integration or version must be provided.')
```

Comment:
Noticed single quotes here. We'll want to make sure that your editor is setup do obey all the linting/formatting rules with ESLint and Prettier, otherwise CI builds will fail (not sure if it will on this project, but definitely in the main project).